### PR TITLE
Delegate removal: added remarks about equality

### LINKED ISSUE #12552: Clarify add and remove delegate operations
See https://github.com/dotnet/samples/pull/929#issuecomment-496512821

Customers have had questions on the details of how delegate addition and subtraction (add and removal).  

/cc @pkulikov

### PR DESCRIPTION
**Pull request by pkulikov**
Fixes #12552 
Supported by dotnet/samples#945

### CODE REVIEW

File: docs/csharp/language-reference/operators/subtraction-operator.md [hunk 1]
```diff
@@ -25,12 +25,22 @@ For information about the arithmetic `-` operator, see the [Unary plus and minus
...
-- If the invocation list of the second operand is not a proper contiguous sublist of the invocation list of the first operand, the result of the operation is the first operand.
+
+  [!code-csharp-interactive[delegate removal](~/samples/csharp/language-reference/operators/SubtractionOperator.cs#DelegateRemoval)]
+
+- If the invocation list of the second operand is not a proper contiguous sublist of the invocation list of the first operand, the result of the operation is the first operand. For example, removing a delegate that is not part of the multicast delegate does nothing and results in the unchanged multicast delegate.
+
+  [!code-csharp-interactive[delegate removal with no effect](~/samples/csharp/language-reference/operators/SubtractionOperator.cs#DelegateRemovalNoChange)]
```
Comment (pkulikov) [inline docs/csharp/language-reference/operators/subtraction-operator.md]: Curious why the build report doesn't complain about missing snippet.

**Comment by BillWagner** [comment]
closing and reopening to start a new build while reviewing.

File: docs/csharp/language-reference/operators/subtraction-operator.md (hunk 1 above)
Comment (BillWagner) [reply docs/csharp/language-reference/operators/subtraction-operator.md]: That's a known issue, and there's an engineering item to track it.

**Review by BillWagner — APPROVED** [review]
Thanks for adding this clarification @pkulikov It will benefit a lot of developers.

I've reviewed the changes, and checked the build, and I'll :shipit: now.

**Pull request merged**